### PR TITLE
tp-qemu: ovs_qos: Improve the error info when test failed

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

### CODE REVIEW

**Comment by CongSmile** [comment]
id: 1203553

File: qemu/tests/ovs_qos.py [hunk 1]
```diff
@@ -80,12 +80,12 @@ def get_throughout(netperf_server, server_vm, netperf_client,
                       logging.info)
         server_ip = server_vm.get_address()
         output = netperf_client.start(server_ip, client_options)
+        logging.info("netperf client output: %s" % output)
```
Comment (ypu) [inline qemu/tests/ovs_qos.py]: We'd better not put the netperf output in the info log, maybe debug log is more suitable

Comment (CongSmile) [inline qemu/tests/ovs_qos.py]: Hi ypu,

Thanks for your review.
Your suggestion is reasonable.I have updated code. 
Could you help check it again?

Thanks.

**Comment by ypu** [comment]
LGTM

Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by FengYang** [comment]
Acked-by: FengYang

**Pull request merged**